diktat-gradle-plugin/pom.xml

```diff
@@ -62,18 +62,39 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <inherited>false</inherited>
                 <executions>
+                    <execution>
+                        <id>gradle-test</id>
+                        <phase>test</phase>
+                        <configuration>
+                            <executable>${gradle.executable}</executable>
+                            <arguments>
+                                <argument>clean</argument>
+                                <argument>jacocoTestReport</argument>
```

Comment:
why do you need test report here? For test coverage?

Reply:
Yes, this tasks will execute `test` and then generate jacoco report.